conf/standard.config

```diff
@@ -0,0 +1,73 @@
+process {
+    withName:
+        "EXPORT_RAW_GENOMES" {
+            container = "amd64/ubuntu"
+            cpus = 1
+            memory = "2 GB"
+    }
+
+    withName:
+        "FASTQC_ORIGINAL|FASTQC_TRIMMED" {
```

Comment:
@Mxrcon , you could use `FASTQC_*` to accomodate both.

Reply:
@abhi18av, I agree, but id like to have more explicit and avoid using wildcard characters so with the addition of new modules I wouldn't get a wrong configuration, what do you think about that?

Reply:
Good point!

However, I think it's safe to assume that anyone who wishes to create an alias for the `FASTQC_` process, would use that prefix.

Reply:
hmm ok!, I'll keep the alias simple and use wildcard, but just for fastqc and multiqc, i'd like to keep this pattern for mtbseq